<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceRecommend</class>
 <widget class="QMainWindow" name="PlaceRecommend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1181</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="school_map">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>591</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="prensent_img">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>50</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前选择为：</string>
      </property>
     </widget>
     <widget class="QLabel" name="present_text">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>220</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>无</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>400</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>自动选择</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>490</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 红色圆形按钮 */
QPushButton {
    border-radius: 12px;          /* 圆角保持圆形 */
    min-width: 20px;            /* 尺寸可根据需求调整 */
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
    background-color: #ff4444;   /* 主背景色（红色） */
    border: 2px solid #cc0000;   /* 边框色（深红） */
    color: white;                /* 文字颜色（确保对比度） */
}

/* 悬停时变深红 */
QPushButton:hover {
    background-color: #cc0000;
}

/* 按下时更深的红 */
QPushButton:pressed {
    background-color: #aa0000;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>460</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>五四体育场</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 红色圆形按钮 */
QPushButton {
    border-radius: 12px;          /* 圆角保持圆形 */
    min-width: 20px;            /* 尺寸可根据需求调整 */
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
    background-color: #ff4444;   /* 主背景色（红色） */
    border: 2px solid #cc0000;   /* 边框色（深红） */
    color: white;                /* 文字颜色（确保对比度） */
}

/* 悬停时变深红 */
QPushButton:hover {
    background-color: #cc0000;
}

/* 按下时更深的红 */
QPushButton:pressed {
    background-color: #aa0000;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>未名湖步道</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>350</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 红色圆形按钮 */
QPushButton {
    border-radius: 12px;          /* 圆角保持圆形 */
    min-width: 20px;            /* 尺寸可根据需求调整 */
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
    background-color: #ff4444;   /* 主背景色（红色） */
    border: 2px solid #cc0000;   /* 边框色（深红） */
    color: white;                /* 文字颜色（确保对比度） */
}

/* 悬停时变深红 */
QPushButton:hover {
    background-color: #cc0000;
}

/* 按下时更深的红 */
QPushButton:pressed {
    background-color: #aa0000;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>理教环路</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>480</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>我选好了</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 红色圆形按钮 */
QPushButton {
    border-radius: 12px;          /* 圆角保持圆形 */
    min-width: 20px;            /* 尺寸可根据需求调整 */
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
    background-color: #ff4444;   /* 主背景色（红色） */
    border: 2px solid #cc0000;   /* 边框色（深红） */
    color: white;                /* 文字颜色（确保对比度） */
}

/* 悬停时变深红 */
QPushButton:hover {
    background-color: #cc0000;
}

/* 按下时更深的红 */
QPushButton:pressed {
    background-color: #aa0000;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>静园周边</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 红色圆形按钮 */
QPushButton {
    border-radius: 12px;          /* 圆角保持圆形 */
    min-width: 20px;            /* 尺寸可根据需求调整 */
    min-height: 20px;
    max-width: 20px;
    max-height: 20px;
    background-color: #ff4444;   /* 主背景色（红色） */
    border: 2px solid #cc0000;   /* 边框色（深红） */
    color: white;                /* 文字颜色（确保对比度） */
}

/* 悬停时变深红 */
QPushButton:hover {
    background-color: #cc0000;
}

/* 按下时更深的红 */
QPushButton:pressed {
    background-color: #aa0000;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>朗润园-镜春园</string>
      </property>
     </widget>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>311</width>
        <height>591</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>school_map</zorder>
     <zorder>prensent_img</zorder>
     <zorder>label</zorder>
     <zorder>present_text</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>label_2</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>label_4</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton_8</zorder>
     <zorder>pushButton_9</zorder>
     <zorder>label_6</zorder>
     <zorder>pushButton_10</zorder>
     <zorder>label_7</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>请选择具体信息：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>230</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>获取推荐地点</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>300</y>
        <width>281</width>
        <height>231</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>56</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>就选这儿</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>180</y>
          <width>56</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>我再想想</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>加载出错</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>天气：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>晴天</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>多云</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>小雨</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>大风</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>星期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>周一</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周二</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周三</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周四</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周五</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周六</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周日</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>时段：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <item>
            <property name="text">
             <string>清晨</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>上午</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>下午</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>傍晚</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>夜间</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="background_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>2</y>
        <width>1061</width>
        <height>611</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>background_2</zorder>
     <zorder>label_3</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>stackedWidget_2</zorder>
     <zorder>layoutWidget</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
